packages/core/saved-objects/core-saved-objects-common/src/saved_objects.ts

```diff
@@ -32,6 +32,7 @@ export type SavedObjectAttribute = SavedObjectAttributeSingle | SavedObjectAttri
  *
  * @public
  * @deprecated This type reduces the type safety of your code. Create an interface for your specific saved object type or use `unknown` instead.
+ * @removeBy 9.0.0
```

Comment:
I added a removeBy version to encourage type-owners to implement their interfaces. If there is a 9.0.0 version, we'll probably have a bunch more deprecations to worry about and shouldn't be carrying existing ones over to a new major.